apps/analytics/src/scripts/6_initial_activity_progress.py

```
@@ -0,0 +1,79 @@
+
+# Script settings
+verbose = False
+
+
+# Fetch all courses from the database
+df_courses = get_running_past_courses(db)
+
+# Iterate over the course and fetch all question responses linked to it
+for idx, course in df_courses.iterrows():
+    course_id = course["id"]
+    print(f"Processing course", idx, "of", len(df_courses), "with id", course_id)
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)